### DISCUSSION TITLE
event node: {"type":"error","status":403,"message":"Forbidden"}

### DISCUSSION THREAD

**Discussion by totti1171** [alexkn/node-red-contrib-homeconnect#33] [Q&A]
Hi,

the authenticification is done,
I can read the devices and the HID - but when I use the event-node, then "{"type":"error","status":403,"message":"Forbidden"}"

![Flow](https://user-images.githubusercontent.com/42803032/108592554-9fd40080-736e-11eb-9dc7-205d3e43d7f7.jpg)
![Node](https://user-images.githubusercontent.com/42803032/108592555-a2365a80-736e-11eb-9cfd-6095591e90b6.jpg)

**Comment by molodeztom** [comment]
I am working on the same problem. Let you know if I find anything.

**Comment by alexkn** [comment]
Mostly it's something wrong with the scopes.
Unfortunately, the eventsource component used in the event node, is not putting the response body into the error object.
And I don't see an easy way to get that without modifying or replacing eventsource.

**Comment by mintgroen** [comment]
My application was running fine (for a year or so) and suddenly I have this error.

{"type":"error","status":403,"message":"Forbidden"}"

Authentication works and the oauth gives a ready status.

Any ideas how to troubleshoot this?

**Comment by shred** [comment]
It was working for me when I added `IdentifyAppliance` as scope, and then reauthenticated. My scope was then `Washer IdentifyAppliance`. I don't know if this was just a coincidence or the solution, though.

**Comment by cernapa** [comment]
Hi all, I got the same message, apparently the "Scope"requirement was added at some point from HomeConnect so now it is required. 

I have added "IdentifyAppliance Monitor Control" as space separated Scope fields as recommended in homeconnect developer docs https://api-docs.home-connect.com/authorization/#authorization-code-grant-flow

<img width="871" alt="Screenshot 2023-09-13 at 18 23 52" src="https://github.com/alexkn/node-red-contrib-homeconnect/assets/31221282/3dbb4abd-aba7-4ab2-a960-afbbb04f4e31">

here is my nodered

<img width="695" alt="Screenshot 2023-09-13 at 18 14 43" src="https://github.com/alexkn/node-red-contrib-homeconnect/assets/31221282/1d2b4f62-3058-410b-829e-afc186530a5e">
My other issue was I have changed the email address that I originally used for login, so when I changed it back everything seem to come back to nomal.
